Explorer/Assets/DCL/SDKComponents/MediaStream/Systems/UpdateMediaPlayerSystem.cs

```diff
@@ -101,7 +101,7 @@ private void UpdateVideoStream(ref MediaPlayerComponent component, PBVideoPlayer
 
             if (component.State != VideoState.VsError)
             {
-                float actualVolume = sdkComponent.Volume * worldVolumePercentage * masterVolumePercentage;
+                float actualVolume = sdkComponent.HasVolume ? sdkComponent.Volume : MediaPlayerComponent.DEFAULT_VOLUME * worldVolumePercentage * masterVolumePercentage;
```

Comment:
idem here?

Reply:
what do you mean? x 2

Reply:
ah sorry, the name might not be the best, the value is a 0->1 float